src/coreclr/tools/aot/crossgen2/crossgen2.props

```diff
@@ -13,12 +13,11 @@
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
     <Configurations>Debug;Release;Checked</Configurations>
-    <ServerGarbageCollection>true</ServerGarbageCollection>
-    <EventSourceSupport>true</EventSourceSupport>
-    <OptimizationPreference>Speed</OptimizationPreference>
     <RunAnalyzers>false</RunAnalyzers>
   </PropertyGroup>
 
+  <Import Project="../AotCompilerCommon.props" />
```

Comment:
Is it OK that crossgen2 is now using `<TieredCompilation>false</TieredCompilation>` when it wasn't before?

Reply:
We don't ship this way (it ships on top of NativeAOT), but this affects inner loop. Disabling tiering is going to be a improvement for inner loop: #83112